lib/response.js

```diff
@@ -264,13 +264,13 @@ module.exports = {
     if (this.ctx.accepts('html')) {
       url = escape(url);
       this.type = 'text/html; charset=utf-8';
-      this.body = `Redirecting to <a href="${url}">${url}</a>.`;
+      this.res.end(`Redirecting to <a href="${url}">${url}</a>.`);
```

Comment:
if you just want to skip all the logic in `this.body=`, maybe use `this._body=` is much better than `this.res.end`, it can break people's code if people want to set some header after this.redirect like this:

```js
app.use(async (ctx, next) => {
  await next;
  ctx.set('cache-control', 'maxage=600');
})
```